spec/models/boarding_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Boarding do
+  let(:boarding) { FactoryBot.build(:boarding) }
```

Comment:
Do you think FactoryBot.create would work here? 🤔

Reply:
Yes apu, `FactoryBot.create` will work here.
The `FactoryBot.build` method is building a new object of the `Boarding` class using the defined factory, but it does not persist the object in the database. On the other hand, FactoryBot.create not only builds the object but also saves it to the database, making it available for future use.